### DISCUSSION TITLE
Almagest skyculture: M7 or NGC6441?

### DISCUSSION THREAD

**Discussion by sunshuwei** [Stellarium/stellarium#1253] [Q&A] [answered]
Almagest skyculture：
"The nebulous star to the rear of the sting [of Scorpion, outside the constellation]" seems to be M7,not NGC6441.
It was first recorded by the 2nd-century Greek-Roman astronomer Ptolemy, who described it as a nebula in 130 AD.

**Comment by sushoff** [comment, marked as answer]
"seems to be M7,not NGC6441" sorry: no, definitely not. NGC6441 would be outshone by the 3 mag star G Sco. 
In principle, it could be G Sco ... the standard identification of M7 was made because of coordinates plus (I guess) the nebulous appearance of a cluster plus

**Comment by sunshuwei** [comment]
> 
> 
> "seems to be M7,not NGC6441" sorry: no, definitely not. NGC6441 would be outshone by the 3 mag star G Sco.
> In principle, it could be G Sco ... the standard identification of M7 was made because of coordinates plus (I guess) the nebulous appearance of a cluster plus

Thank you。
